docs/topics/dokka-plugins.md

```diff
@@ -13,11 +18,11 @@ If you want to learn how to create Dokka plugins, see
 
 ## Apply Dokka plugins
 
-Dokka plugins are published as separate artifacts, so to apply a Dokka plugin you only need to add it as a dependency.
-From there, the plugin extends Dokka by itself - no further action is needed.
+Dokka plugins are published as separate artifacts, so to apply a Dokka plugin, you only need to add it as a dependency. 
+From there, the plugin extends Dokka by itself—no further action is needed.
```

Comment:
```suggestion
From there, the plugin extends Dokka by itself – no further action is needed.
```

En dash instead of em dash, and we generally do use spaces around them.